src/api/java/com/minecolonies/api/client/render/modeltype/BipedModelType.java

```diff
@@ -15,6 +15,7 @@
     // Lumberjack: 4 male, 1 female
     LUMBERJACK("lumberjack", 1),
     FARMER("farmer", 1),
+    GRAVEDIGGER("gravedigger", 1),
     FISHERMAN("fisherman", 1),
```

Comment:
We talked about this the other day, and I think "Undertaker" is the best name for the job (Graveyard will stay for the building)

Reply:
I associate undertakers more with funerals, and it doesn't look like those get added. I still prefer Gravedigger

Reply:
I actually like "Undertaker" as they are generally responsible for all arrangements, with the gravedigger just being a subcontractor for the actual digging. Additionally, with the shortened mourning time, if we could move mourning to graveside when there is a grave the "Undertaker" effectively *is* running the funeral.

Reply:
What is the consensus here? Should I rename the gravedigger to Undertaker?

Reply:
ye because he does more than just gravedigging

Reply:
Ok I'll do it then 😄